firefox-ios/Client/Frontend/Autofill/Address/Edit/UpdatableAddressFields+Decodable.swift

```diff
@@ -0,0 +1,53 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Storage
+import struct MozillaAppServices.UpdatableAddressFields
+
+extension UpdatableAddressFields: Decodable {
```

Comment:
Is this coming from A~S?
Curious if we can ask AS to make it codable there

Reply:
Yess. That would be helpful since they already make it equatable. One issue that might arise is the decoding strategy since we see it here using the kebab case. In some context we might want it to be snake case? Or change the decoding keys.